### PR TITLE
Completed css-brochure-form exercise

### PR DESCRIPTION
**Pull request by wang-morris**
Screen Shot 2022-05-19 at 9 04 54 PM

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+input {
+  padding: 8px 270px 8px 8px;
+  border-radius: 5px;
+  border-color: lightgrey;
+  margin-bottom: 10px;
+}
+
+select {
+  padding: 8px 300px 8px 8px;
```
Comment (BrettAlbright) [inline css-brochure-form/styles.css]: try to avoid using large amount of padding like this.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+label {
+  font-weight: 500;
+  font-size: 12px;
+  margin-left: 10px;
+}
+
+input {
+  padding: 8px 270px 8px 8px;
```
Comment (BrettAlbright) [inline css-brochure-form/styles.css]: same as below

File: css-brochure-form/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      <div>
+        <label for="name-input">NAME</label>
+      </div>
+      <input type="text" name="name-input" id="name-input" placeholder="Sally Smith">
+      <div>
+        <label for="location">WHERE ARE YOU HIRING?</label>
+      </div>
+      <input type="text" name="location" id="location" placeholder="City, Country">
+      <div>
+        <label for="dropdown">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT 6 MONTHS?</label>
```
Comment (scrumptiousdev) [inline css-brochure-form/index.html]: Any thought on `for` attribute value?

File: css-brochure-form/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h1>LEARN MORE</h1>
```
Comment (BrettAlbright) [inline css-brochure-form/index.html]: I would wrap all of the content in a `container` element with a set width and then center that on the page.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Looking good overall! make sure to add the border around the inputs to match the example, and update based on the other feedback given. Reach out if you have questions!

File: css-brochure-form/styles.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  margin-bottom: 10px;
+  background: rgb(0 0 0 / 5%);
+}
+
+button {
+  background-color: #eb6534;
+  border-radius: 15px;
+  padding: 3px 15px;
+  color: white;
+  margin-left: 345px;
```
Comment (scrumptiousdev) [inline css-brochure-form/styles.css]: After [wrapping with `.container`,](https://github.com/wang-morris/m1-0522-code-solutions/pull/16/files#r877713872) try not to use `margin-left` to have button aligned to the right

**Comment by wang-morris** [comment]
Screen Shot 2022-05-20 at 3 19 28 PM

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**